packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -636,6 +662,28 @@ abstract class Page<T> extends RouteSettings {
   ///    Flutter.
   final String? restorationId;
 
+  /// Called after a pop on the associated route was handled.
+  ///
+  /// It's not possible to prevent the pop from happening at the time that this
+  /// method is called; the pop has already happened. Use [canPop] to
+  /// disable pops in advance.
+  ///
+  /// This will still be called even when the pop is canceled. A pop is canceled
+  /// when the associated [Route.popDisposition] returns false, or when
+  /// [canPop] is set to false. The `didPop` parameter indicates whether or not
+  /// the back navigation actually happened successfully.
+  final PopInvokedCallback onPopInvoked;
+
+  /// When false, blocks the associated route from being popped.
+  ///
+  /// This includes the root route, where upon popping, the Flutter app would
+  /// exit.
+  ///
+  /// If there are also [PopScope] widgets appear in a route's widget subtree,
+  /// each of their `canPop` must be `true` in order for the route to be
+  /// able to pop.
```

Comment:
Maybe add this clarification: "each of their `canPop` must be `true`, in addition to this `canPop`, in order for the route to be able to pop."